libnmstate/schemas/operational-state.yaml

```diff
@@ -456,3 +470,27 @@ definitions:
         type: array
         items:
           type: string
+  vlan-filter:
+    type: object
+    properties:
+      id:
+        type: integer
```

Comment:
Can we use a yaml reference here to reuse type ?

Reply:
sure, nice suggestion.